lib/screens/CourseDirectoryScreen.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:we_find/model/modelTranslate.dart';
+
+import 'StudyModuleScreen.dart';
+
+// TODO: This is just a copy of StudyModuleScreen with some changes
+// if you know how to fix this feel free.
+
+class CourseDirecotryScreen extends StatefulWidget {
+  final StudyModuleTranslate _studyModule;
+  const CourseDirecotryScreen(this._studyModule);
+
+  @override
+  _CourseDirecotryScreenState createState() => _CourseDirecotryScreenState();
+}
+
+class _CourseDirecotryScreenState extends State<CourseDirecotryScreen> {
+  String _trimName(String name) {
+    if (name.startsWith("Studienprogrammleitung") ||
+        name.startsWith("Directorate of Studies"))
+      return name.substring(name.indexOf(" ") + 1);
+    return name;
+  }
+
+  ListView _buildStudyModuleScreen(ThemeData themeData) {
+    List<Widget> view = [];
+
+    view.add(Text(
+      "Course Directory", // TODO: LANGUAGE
+      style: themeData.textTheme.headline2,
+    ));
+    view.add(Divider());
+
+    for (StudyModuleTranslate eachStudyModule
```

Comment:
nitpick but let's rename `StudyModuleTranslate` to something else. It evokes almost a method name for me, let's go with `StudyModuleTranslated` or `StudyModuleInternationalised` or something similar.

Reply:
Maybe the choice of name really is poor as it is not supposed to handle translating. It's more of an abstraction layer between the actual `StudyModule`, which contains optionals and the actual app, to make accessing the widgets a bit easier.

Maybe `StudyModuleWrap` or `StudyModuleAbstract`?

Reply:
Maybe `StudyModuleDisplay` if it handles displayable data?

Reply:
Hmmm I don't think I like that too much. It does not handle displaying the data rather just preparing it for displaying.

`StudyModuleParsed` maybe?

Reply:
`StudyModuleWrapped` <- final name